styx-scheduler-service/src/main/java/com/spotify/styx/StyxScheduler.java

```diff
@@ -435,7 +436,6 @@ private void setupMetrics(
       Stats stats) {
 
     final Gauge<Long> queuedEventsCount = stateManager::getQueuedEventsCount;
-    final Gauge<Long> activeStatesCount = stateManager::getActiveStatesCount;
```

Comment:
The same graph achieved by the old one can still be constructed from the new one I believe